mesh/ncmesh.hpp

```diff
@@ -331,6 +331,7 @@ class NCMesh
 
    /// Return the number of root elements.
    int GetNumRootElements() { return root_state.Size(); }
+   virtual int GetNumGhostElements() const { return 0; }
```

Comment:
Please don't add this. In fact please remove `ParNCMesh::GetNumGhostElements` and `ParNCMesh::GetNumGhostVertices`  for me (I forgot to get rid of those in `par-ncmesh-saving-dev`).